assets/translations/zh-hans.yaml

```diff
@@ -0,0 +1,1801 @@
+##############################
+# Special stuff
+##############################
+generic:
+  add: 添加
+  back: 返回
+  cancel: 取消
+  clickToShow: "[显示值]" 
+  close: 关闭
+  comingSoon: 即将推出
+  create: 创建
+  created: 创建时间 
+  customize: 定制 
+  default: 默认
+  disabled: 禁用
+  enabled: 启用
+  enforced: Enforced # no use
```

Comment:
If you run into something you're sure is unused you can just remove it from English (this is a good example).